guides/common/modules/con_upgrading-el-on-project-or-proxy.adoc

```diff
@@ -8,6 +8,7 @@ There are two methods to upgrade your {Project} or {SmartProxy} from {EL} 8 to {
 
 * Leapp in-place upgrade
 * Migration by using backup and restore
+* Migration using cloning
```

Comment:
"by using" or "using"? I fail at such constructs :see_no_evil:

Reply:
I always look at https://github.com/theforeman/foreman-documentation/issues/3141 again. I still get it wrong here and there. :see_no_evil: